src/components/ProductSlider/ProductSlider.tsx

```diff
@@ -8,6 +8,8 @@ import styles from './style.module.scss';
 import { ProductCard } from '../ProductCard/ProductCard';
 import { Product } from '@/types/Product';
 import { IconButton } from '../IconButton';
+import { useProducts } from '@/hooks';
+import SkeletonGrid from '../Skeleton/Skeleton';
```

Comment:
```suggestion
import SkeletonGrid from '../Skeleton';
```
We have index file. It would be nice to fix it above as well